permissions/plugin-aws-codecommit-trigger.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "aws-codecommit-trigger"
+paths:
+  - "org/jenkins-ci/plugins/aws-codecommit-trigger"
```

Comment:
@daniel-beck I've just updated my email address on the account. Thank you for your help!